addon.xml

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="metadata.tvdb.com.python" name="The TVDB (new)" version="1.0.3" provider-name="Team Kodi">
+<addon id="metadata.tvdb.com.python" name="The TVDB (new)" version="1.0.4" provider-name="Team Kodi">
   <requires>
     <import addon="xbmc.metadata" version="2.1.0" />
     <import addon="xbmc.python" version="3.0.0" />
     <import addon="script.module.tvdbsimple" version="1.0.6" />
     <import addon="script.module.trakt" version="3.1.0" />
   </requires>
-  <extension point="xbmc.metadata.scraper.tvshows" language="en" library="scraper.py" />
+  <extension point="xbmc.metadata.scraper.tvshows" language="en" library="scraper.py" cachepersistence="00:15" />
```

Comment:
I have doubts that this option has any effect on Python scrapers.

Reply:
It works for any kind of scraper.